assets/styles/themes/_light.scss

```diff
@@ -311,7 +311,7 @@ BODY, .theme-light {
   --header-height              : 55px;
   --header-border              : #{$medium};
   --header-border-size         : 1px;
-  --nav-width                  : 230px;
+  --nav-width                  : 260px;
```

Comment:
@richard-cox , Yes, because `Virtualization Management` requires a large width, if you don’t want to change this width, I think the dashboard team can help see how to modify the UI style design. We will delete this 260px modification and rebase upstream branch changes。   What do you think?   

cc @n313893254

Reply:
I will do it.

Reply:
Resolved via https://github.com/harvester/dashboard/pull/101